ansible/roles/cli/tasks/download_cli_linux.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- name: "ensure cli sub directory structure exists"
+  file:
+    path: "{{ cli_nginx_dir }}/linux/amd64"
+    state: directory
+
+- name: "download cli into ngnix"
+  local_action: >
+    get_url
+    url="https://github.com/openwhisk/openwhisk-cli/releases/download/latest/OpenWhisk_CLI-latest-linux-amd64.zip"
+    dest="{{ cli_nginx_dir }}/linux/amd64/OpenWhisk_CLI-linux.zip"
+    validate_certs=False
+
+- name: "unarchive cli into ngnix"
+  unarchive: >
+    src="{{ cli_nginx_dir }}/linux/amd64/OpenWhisk_CLI-linux.zip"
+    dest="{{ cli_nginx_dir }}/linux/amd64"
+    creates=wsk
+    mode=0755
+
+- name: "download cli (linux) to openwhisk home at {{ openwhisk_home }}"
+  local_action: >
+    get_url
+    url="https://{{ groups['edge'] | first }}/cli/go/download/linux/amd64/wsk"
+    dest="{{ openwhisk_home }}/bin/wsk"
+    mode=0755
+    validate_certs=False
```

Comment:
That file and the mac download file are close to exact copies. Can we parametrize? You can have a look at the db scripts for inspiration.